eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/processors/SubSnpNoHgvsToDbsnpVariantsWrapperProcessor.java

```diff
@@ -43,8 +45,16 @@
 
 public class SubSnpNoHgvsToDbsnpVariantsWrapperProcessor implements ItemProcessor<SubSnpNoHgvs, DbsnpVariantsWrapper> {
 
+    private static final String DECLUSTERED_ALLELES_AND_TYPE_MISMATCH = "Declustered (Alleles and type mismatch)";
+
+    private static final String DECLUSTERED_ALLELES_MISMATCH = "Declustered (Alleles mismatch)";
```

Comment:
I think that "Declustered: None of the variant alleles match the reference allele" is more informative (as was suggested in other PR). If this message is updated, the other messages should be updated too.